go.mod

```diff
@@ -1,46 +1,96 @@
 module github.com/logzio/metric-stream-lambda
 
-go 1.17
+go 1.22.0
```

Comment:
Seems like the tests fail because of the version here

Reply:
its sorted now :)